raiden/transfer/channel.py

```diff
@@ -1693,22 +1802,44 @@ def handle_action_set_fee(
 
 def handle_action_withdraw(
     channel_state: NettingChannelState,
-    withdraw: ActionChannelWithdraw,
+    action_withdraw: ActionChannelWithdraw,
     pseudo_random_generator: random.Random,
+    block_number: BlockNumber,
 ) -> TransitionResult[NettingChannelState]:
-    is_valid, msg = is_valid_action_withdraw(channel_state, withdraw)
+    events: List[Event] = list()
+    is_valid, msg = is_valid_action_withdraw(channel_state, action_withdraw)
 
     if is_valid:
-        channel_state.our_state.total_withdraw = withdraw.total_withdraw
-        events: List[Event] = send_withdraw_request(
+        nonce = get_next_nonce(channel_state.our_state)
+        # Calculate the withdraw's expiry exactly the same as we
```

Comment:
IMO this should be part of the function docstring, it is relevant for both the expirations